Add Glimmer build output to CI/CD.

rwjblue opened the pull request.

Commit:
Add Glimmer build output to CI/CD.

chancancode commented on ember-cli-build.js:
```diff
@@ -33,6 +33,10 @@ function babelConfigFor(environment) {
   }
 
+  if (process.env.EMBER_GLIMMER) {
+    features['ember-glimmer'] = true;
```
should we set everything else to false?

rwjblue replied:
yes, will do

chancancode commented on bin/bower_ember_build:
```diff
@@ -39,16 +39,17 @@ case $TRAVIS_BRANCH in
+fi
+
 # Sending output to /dev/null to prevent GH_TOKEN leak on error.
 git clone --branch ${CHANNEL} https://${USER}:${GH_TOKEN}@github.com/${COMPONENTS_EMBER_REPO_SLUG}.git bower_ember &> /dev/null
```
Wouldn't this fail since glimmer is not a real branch? Maybe do the override after here?

rwjblue replied:
It is a real branch

rwjblue replied:
https://github.com/components/ember/tree/glimmer

chancancode replied:
I see 👌

Commit:
Refactor feature and package gathering processes.

Also adds a cache key for invalidating when `features.json`
changes (still requires a restart).

rwjblue commented:
I've done more work here (large amount of the changes are whitespace specific), this is ready for another quick pass but not ready to land.

rwjblue commented:
Just rebased...
